expansion.cpp

```diff
@@ -513,6 +513,7 @@ void celldrawer::do_viewdist() {
         celldistance(c, distance_from == dfPlayer ? cwt.at : currentmap->gamestart());
       dc = (d != cd) ? 0xFF0000 : 0x00FF00;
       label = its(d);
+      break;
       }
     case ncNone: ;

```

Comment:
Consider adding a `break;` here as well. If another `case` were added after this one, and you forgot to add the `break;` in this one, then you'd have a bug. Fortunately, once we enable `-Wimplicit-fallthrough`, the compiler will catch that particular bug and so it doesn't really matter whether you add the `break;` here now or wait until the compiler tells you it's needed.